UmpleToRTCpp/UmpleTLTemplates/Attributes.ump

```diff
@@ -24,6 +24,24 @@ association { 0..1 AbstractGenConstraint constraint -- * GenConstraintCondition
 association { 0..1 AbstractGenConstraint listenerOwner -- * GenConstraintVariable listeners;}
 
 class GenMethod{
+
+    boolean equals(Object object) {
```

Comment:
In our discussion, I mentioned that comparing methods is by signature, not only by name. The check against name was only one out of many. You still need to check for param types. After that, ensure this did not cause the fix to stop working.